mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioNavigator.tsx

```diff
@@ -240,29 +249,61 @@ export function PortfolioNavigator (): JSX.Element {
       />
 
       <PortfolioStack.Screen
-        component={SendScreen}
+        component={isFeatureAvailable('send_v2') ? SendScreenV2 : SendScreen}
         name='Send'
         options={{
+          ...screenOptions,
           headerTitle: () => (
             <HeaderTitle
```

Comment:
v2 header title doesnt use `<HeaderTitle>` anymore, might need to add conditonal statement here, same thing for send confirm screen